tests/build.go

```diff
@@ -116,6 +116,15 @@ func Build(o *option.Option) {
 				gomega.Expect(stdErr).Should(gomega.ContainSubstring("progress flag set:2"))
 			})
 
+			ginkgo.It("build image with --output flag", func() {
+				outputFilePath := filepath.Join(buildContext, "out.tar")
+				dest := fmt.Sprintf("type=tar,dest=%s", outputFilePath)
+				command.Run(o, "build", "-t", "output:tag", "--output", dest, buildContext)
+				// When --output flag is enabled build artifacts exported as files and not as a local image.
+				imageShouldNotExist(o, "output:tag")
+				gomega.Expect(ffs.CheckIfFileExists(outputFilePath)).To(gomega.Equal(true))
```

Comment:
Instead of only checking if there's a file, probably we can `load` the tar into image store to verify that the thing in the tar makes sense? Something like:

```go
imageShouldNotExist(o, "output:tag")
// load the tar
imageShouldExist(o, "output:tag")
```

Reply:
This makes sense. However there is a [bug](https://github.com/containerd/nerdctl/issues/1804) in `nerdctl` which does not allow me to build the image in `oci` or `docker` format.  Also nerdctl currently does not implement `import`  so I can't import the "filesystem" tar file exported by build. See https://github.com/containerd/nerdctl/issues/908 and https://github.com/containerd/nerdctl#unimplemented-docker-commands

Reply:
I thought `load` rather than `import` should be used here: https://stackoverflow.com/a/36932570/6355435

Reply:
I actually tried that but hit this [error/bug](https://github.com/moby/moby/issues/19566#issuecomment-316839687). 
Snippet 
```
% cat Dockerfile
FROM public.ecr.aws/docker/library/alpine:3.13
siravara@bcd074105a94 vishwas-tests % docker build -t output:tag --output type=tar,dest=docker.tar .
[+] Building 1.0s (5/5) FINISHED                                                    
 => [internal] load build definition from Dockerfile                           0.0s
 => => transferring dockerfile: 89B                                            0.0s
 => [internal] load .dockerignore                                              0.0s
 => => transferring context: 2B                                                0.0s
 => [internal] load metadata for public.ecr.aws/docker/library/alpine:3.13     0.8s
 => CACHED [1/1] FROM public.ecr.aws/docker/library/alpine:3.13@sha256:469b6e  0.0s
 => exporting to client                                                        0.1s
 => => sending tarball                                                         0.1s
siravara@bcd074105a94 vishwas-tests % docker load < docker.tar 
open /var/lib/docker/tmp/docker-import-2606953934/bin/json: no such file or directory
```

Reply:
Thanks for the investigation. Maybe we add the following TODO and merge this one then.

```go
// TODO: Test if we can `import` the tar ball after `nerdctl import` is supported.
```

Reply:
Thanks for the review. Added TODO comment.